spec/automation/unit/method_validation/validate_quota_spec.rb

```diff
@@ -1,7 +1,7 @@
 describe "Quota Validation" do
   include Spec::Support::QuotaHelper
 
-  def run_automate_method(attrs, used, requested)
+  def run_automate_method(_attrs, used, requested)
```

Comment:
Do we need to keep it as an unused argument?

Reply:
Thanks Lucy, that's a good point.  @d-m-u Can we remove the arg?